masterlist.yaml

```diff
@@ -2576,6 +2576,13 @@ plugins:
       - name: 'RLO - Interiors.esp'
         condition: 'not file("Realistic Lighting Overhaul - Major City Interiors.esp")'
 
+  - name: 'ENB Light.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/22574' ]
+    after:
+      - 'ELE_SSE.esp'
```

Comment:
It would still require a patch, but I've have no objections to the after.
Bash for SSE doesn't handle light records, so no tags are required at the moment.